modules/machine-autoscaler-cr.adoc

```diff
@@ -24,9 +24,16 @@ spec:
     kind: MachineSet <5>
     name: worker-us-east-1a <6>
 ----
-<1> Specify the machine autoscaler name. To make it easier to identify which machine set this machine autoscaler scales, specify or include the name of the machine set to scale. The machine set name takes the following form: `<clusterid>-<machineset>-<aws-region-az>`
-<2> Specify the minimum number machines of the specified type that must remain in the specified zone after the cluster autoscaler initiates cluster scaling. If running in AWS, GCP, Azure, or {rh-openstack}, this value can be set to `0`. For other providers, do not set this value to `0`.
+<1> Specify the machine autoscaler name. To make it easier to identify which machine set this machine autoscaler scales, specify or include the name of the machine set to scale. The machine set name takes the following form: `<clusterid>-<machineset>-<aws-region-az>`.
+<2> Specify the minimum number machines of the specified type that must remain in the specified zone after the cluster autoscaler initiates cluster scaling. If running in AWS, GCP, Azure, {rh-openstack}, or vSphere, this value can be set to `0`. For other providers, do not set this value to `0`.
 <3> Specify the maximum number machines of the specified type that the cluster autoscaler can deploy in the specified AWS zone after it initiates cluster scaling. Ensure that the `maxNodesTotal` value in the `ClusterAutoscaler` resource definition is large enough to allow the machine autoscaler to deploy this number of machines.
 <4> In this section, provide values that describe the existing machine set to scale.
 <5> The `kind` parameter value is always `MachineSet`.
 <6> The `name` value must match the name of an existing machine set, as shown in the `metadata.name` parameter value.
+
+If your machines are running in AWS, GCP, Azure, {rh-openstack}, or vSphere, you can set the `spec.minReplicas` value to `0`. You can save on costs by setting this value to zero for use cases such as running expensive or limited-usage hardware used for specialized workloads, or scaling a machine set with extra large machines. The machine autoscaler scales the machine set down to zero when the machines are not in use.
```

Comment:
```suggestion
If your machines are running in AWS, GCP, Azure, {rh-openstack}, or vSphere, you can set the `spec.minReplicas` value to `0`. You can save on costs by setting this value to zero for use cases such as running expensive or limited-usage hardware that is used for specialized workloads, or by scaling a machine set with extra large machines. The machine autoscaler scales the machine set down to zero if the machines are not in use.
```

Reply:
Would it make sense to attach this to callout no. 2, or to wrap it in an admonition?

Could probably do some word golfing here if you want. :shrug:

Reply:
@maxwelldb Thank you for the great suggestions. I went back and forth on whether to include the additional context for scaling to/from zero in the callout vs a separate paragraph. The first part of the new paragraph does repeat a part of the callout. I'll move it to the callout and push the change to see how things look.